test/test_mkldnn.py

```diff
@@ -1469,24 +1469,33 @@ def _lstm_params_list(self):
         params_list = list(params_dict.values())
         return params_list
 
-    def _cast_dtype(self, input, bf16):
+    def _cast_dtype(self, input, bf16, fp16):
         if bf16:
             input = input.to(torch.bfloat16)
+        elif fp16:
+            input = input.to(torch.half)
         return input
 
-    @unittest.skipIf(IS_WINDOWS, "Limit support for bf16 path")
     def test_lstm(self):
         seed = 2023
         torch.manual_seed(seed)
 
         params_list = self._lstm_params_list()
         for dtype in types:
             bf16 = True if dtype == torch.bfloat16 and torch.ops.mkldnn._is_mkldnn_bf16_supported() else False
+            fp16 = True if dtype == torch.half and torch.ops.mkldnn._is_mkldnn_fp16_supported() else False
```

Comment:
could maybe be replaced simply by `fp16 = dtype == torch.half and torch.ops.mkldnn._is_mkldnn_fp16_supported()`?

Reply:
Thanks for your comments. Revised.